From 03a8e54200be338c75d753ee02f33c88ff8cd7df Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 19:04:20 -0400
Subject: [fs] btrfs: ioctl - fix typo s/substract/subtract/

Message-id: <1407179109-13714-374-git-send-email-sandeen@redhat.com>
Patchwork-id: 87314
O-Subject: [RHEL7.1 PATCH 373/422] trivial: fs/btrfs/ioctl.c: fix typo s/substract/subtract/
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Antonio Ospite <ao2@ao2.it>

commit 9391558411e89915f7f7c5c996d3091eced1a0ef
Author: Antonio Ospite <ao2@ao2.it>
Date:   Wed Jun 4 14:03:48 2014 +0200

    trivial: fs/btrfs/ioctl.c: fix typo s/substract/subtract/

    Signed-off-by: Antonio Ospite <ao2@ao2.it>
    Cc: Chris Mason <clm@fb.com>
    Cc: Josef Bacik <jbacik@fb.com>
    Cc: linux-btrfs@vger.kernel.org
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/ioctl.c b/fs/btrfs/ioctl.c
index 44dcfd0..dd9a02a 100644
--- a/fs/btrfs/ioctl.c
+++ b/fs/btrfs/ioctl.c
@@ -3216,11 +3216,11 @@ process_slot:
      * | ------------- extent ------------- |
      */
 
-    /* substract range b */
+    /* subtract range b */
     if (key.offset + datal > off + len)
      datal = off + len - key.offset;
 
-    /* substract range a */
+    /* subtract range a */
     if (off > key.offset) {
      datao += off - key.offset;
      datal -= off - key.offset;
-- 
1.7.1